hsi_filter_red:
    use_indices: false
    keep_organized: true
    h_limit_max: 50
    h_limit_min: -50
    s_limit_max: 255
    s_limit_min: 100
    i_limit_max: 255
    i_limit_min: 0

pointcloud_to_laserscan:
    target_frame: ground_plane_in_sensor_frame # rs_d435_color_frame
    transform_tolerance: 0.01
    min_height: 0.0
    max_height: 0.1

    angle_min: -0.5 # less than the end link
    angle_max: 0.5 # less than the end link
    angle_increment: 0.0087 # M_PI/360.0
    scan_time: 10 # ?
    range_min: 0.45
    range_max: 4.0
    use_inf: true
    inf_epsilon: 1.0

    concurrency_level: 0
        
line_extractor:
    scan_topic: /ground_plane_in_sensor_frame/scan
    publish_markers: true
    bearing_std_dev: 0.01
    range_std_dev: 0.012
    least_sq_angle_thresh: 0.1
    least_sq_radius_thresh: 0.1
    max_line_gap: 0.5
    min_line_length: 0.1
    min_range: 0.01
    min_split_dist: 0.1
    outlier_dist: 0.1
    min_line_points: 10

get_horizontal_object_info:
    detected_line_name: /line_markers
    camera_frame_name: rs_d435_color_optical_frame
   